pkg/component/component.go

```diff
@@ -886,6 +886,36 @@ func List(client *occlient.Client, applicationName string) (ComponentList, error
 
 	}
 
+	if len(components) == 0 {
+		localConfig, err := config.New()
```

Comment:
can we extract this out into a separate function? seems like its not dependent on other variables?

Reply:
done